README-Gitpod.md

```diff
@@ -0,0 +1,41 @@
+Token Contract in AssemblyScript - Gitpod version
+=================================================
+
+This is a stripped-down example where we'll demonstrate creating a token. We'll visit a page, sign in and use your browser's console to run commands to initialize, send, and get the balance of a custom token.
+
+This README is specific to Gitpod and this example. For local development, please see [README.md](README.md).
+
+This project contains implementation of token contract similar to [ERC20](https://theethereum.wiki/w/index.php/ERC20_Token_Standard).
+
+Getting started
+---------------
+
+Gitpod has taken care of installing all the necessary tools and dependencies. At the bottom of Gitpod is a terminal which will display a link to follow:
```

Comment:
Maybe "If you're viewing this file in Gitpod, at the bottom of the page you'll see a terminal which will display..."

Reply:
can you please make copy changes? I need to do other things, thanks!

Reply:
Yep! Sorry, missed your note until now